README.md

```diff
@@ -40,46 +40,34 @@ $ sudo apt update
 $ (. /etc/os-release && sudo dnf install -y https://packages.red-data-tools.org/centos/${VERSION_ID}/red-data-tools-release-latest.noarch.rpm)
 ```
 
-## OpenCV GLib
+## How to install packages
 
-This section describes how to install
-[OpenCV GLib](https://github.com/red-data-tools/opencv-glib) package.
-
-### Debian GNU/Linux and Ubuntu
+### OpenCV GLib
 
 ```console
-$ sudo apt install -y -V libopencv-glib-dev
+$ sudo apt install libopencv-glib-dev
 ```
 
-## GR framework
-
-This section describes how to install
-[GR framework](https://gr-framework.org/) package.
-
-### Debian GNU/Linux and Ubuntu
+### GR framework
 
 ```console
-$ sudo apt install -y -V libgrm-dev
+$ sudo apt install libgrm-dev
 ```
 
-### CentOS
```

Comment:
Why do you want to remove platform information?

Reply:
Because this is the development section and I think it will be obvious to most readers that Debian/Ubuntu is APT and CentOS is YUM.